.gitignore

```diff
@@ -26,5 +26,6 @@ target/
 
 # hide client information
 src/main/resources/
+src/main/secrets/
 
```

Comment:
I was planning on putting this into a separate PR tomorrow. Up to you to keep it